sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala

```diff
@@ -284,6 +287,7 @@ object JSONOptions extends DataSourceOptions {
   val TIME_ZONE = newOption("timeZone")
   val WRITE_NON_ASCII_CHARACTER_AS_CODEPOINT = newOption("writeNonAsciiCharacterAsCodePoint")
   val SINGLE_VARIANT_COLUMN = newOption("singleVariantColumn")
+  val USE_UNSAFE_ROW = newOption("useUnsafeRow")
```

Comment:
We should actually document this `docs/sql-data-sources-json.md`, and expose this in Python API too, e.g., `DataFrameReader.json` and `DataFrameWriter.json`

Reply:
@HyukjinKwon Just added to the doc. We don't need to especially expose it. `.option("useUnsafeRow", "true")` will just work. This is also how other options are used.

Reply:
The session config is annotated with `.internal()`, shall we reveal it to end-users too since its companion is public